scripts/bundles/internal.ts

```diff
@@ -1,3 +1,4 @@
+import { generateDtsBundle } from 'dts-bundle-generator/dist/bundle-generator.js';
```

Comment:
There is probably no other way around this but I am afraid this can break at any time. Should we at least raise an issue in `dts-bundle-generator` and ask if they could expose the function from the package?

Reply:
I actually just copied that from our `bundle-dts.ts` script, here:

https://github.com/ionic-team/stencil/blob/97dcb45d44751d239b0afb6380bea217818b211a/scripts/utils/bundle-dts.ts#L1

But I don't think it's necessary to do the relative import like that, since importing those things directly from `dts-bundle-generator` seems to work fine. I'll change it in both locations.